tendrl/commons/flows/expand_cluster/utils.py

```diff
@@ -0,0 +1,58 @@
+import copy
```

Comment:
Isnt this doing the same as you explained https://github.com/Tendrl/commons/blob/develop/tendrl/commons/flows/create_cluster/utils.py#L140